public/app/features/explore/Explore.tsx

```diff
@@ -332,7 +338,8 @@ export class Explore extends React.PureComponent<ExploreProps, ExploreState> {
               <SecondaryActions
                 addQueryRowButtonDisabled={isLive}
                 // We cannot show multiple traces at the same time right now so we do not show add query button.
-                addQueryRowButtonHidden={mode === ExploreMode.Tracing}
+                //TODO:unification
+                // addQueryRowButtonHidden={mode === ExploreMode.Tracing}
```

Comment:
I think we can just show one trace view per frame and deal with multiple traces that way.